package.json

```diff
@@ -40,7 +40,7 @@
     "release": "npm run build && npm run build:package && npm test && ./scripts/release.js",
     "release:dist": "./scripts/release.js",
     "start": "webpack-dev-server --progress",
-    "test": "npm run lint && npm run flow:restart && npm run jest -- --coverage"
+    "test": "npm run lint && npm run flow:restart && npm run jest -- --coverage --runInBand"
```

Comment:
I'm generally not a fan of forcing serialization. I think we ought to do some more tuning here https://blog.bam.tech/developper-news/optimise-jest-in-ci-and-save-hours-a-week-literally-1

Reply:
Is anyone, really?  Travis was hanging otherwise...  Using `maxWorkers=2`seems to work and  have better performance.  I'll make the update.